config/resources/domain.json

```diff
@@ -1,188 +1,150 @@
-              }
-          },
-          "new-resource-base": "http://data.lblod.info/maatregelconcepten/"
+      "relationships": {
+        "status": {
+          "predicate": "vs:term_status",
+          "target": "roadSignConceptStatus",
+          "cardinality": "one"
+        },
+        "categories": {
+          "predicate": "org:classification",
+          "target": "roadSignCategory",
+          "cardinality": "many"
         },
-        "road-sign-concepts": {
-            "name": "roadSignConcept",
-            "class": "mobiliteit:Verkeersbordconcept",
-            "attributes": {
-                "image": {
-                    "type": "url",
-                    "predicate": "mobiliteit:grafischeWeergave"
-                },
-                "meaning": {
-                    "type": "string",
-                    "predicate": "skos:scopeNote"
-                },
-                "road-sign-concept-code": {
-                    "type": "string",
-                    "predicate": "skos:prefLabel"
-                }
-            },
-            "relationships": {
-                "status": {
-                    "predicate": "vs:term_status",
-                    "target": "roadSignConceptStatus",
-                    "cardinality": "one"
-                },
-                "categories": {
-                    "predicate": "org:classification",
-                    "target": "roadSignCategory",
-                    "cardinality": "many"
-                },
-                "measureconcepts": {
-                    "predicate": "lblodmow:verkeersbordconcept",
-                    "target": "measureConcept",
-                    "cardinality": "many",
-                    "inverse": true
-                }
-            },
-            "new-resource-base": "http://data.lblod.info/verkeersbordconcepten/"
+        "sub-sign": {
+          "predicate": "lblodmow:heeftOnderbordConcept",
+          "target": "roadSignConcept",
+          "cardinality": "many"
         },
-        "road-sign-categories": {
-            "name": "roadSignCategory",
-            "class": "mobiliteit:Verkeersbordcategorie",
-            "attributes": {
-                "label": {
-                    "type": "string",
-                    "predicate": "skos:prefLabel"
-                }
-            },
-            "relationships": {
-                "roadSignConcepts": {
-                    "predicate": "org:classification",
-                    "target": "roadSignConcept",
-                    "cardinality": "many",
-                    "inverse": true
-                }
-            },
-            "new-resource-base": "http://data.lblod.info/verkeersbordcategorieen/"
+        "related-road-sign-concepts": {
+          "predicate": "lblodmow:heeftGerelateerdVerkeersbordconcept",
+          "target": "roadSignConcept",
+          "cardinality": "many"
+        }
+      },
+      "new-resource-base": "http://data.lblod.info/verkeersbordconcepten/"
```

Comment:
Unrelated to the other changes in this PR, but should we change this to `http://data.vlaanderen.be/id/concept/Verkeersbordconcept/`? It looks like that's how they are stored in the DB?

Reply:
It's better to ask Niels about the different URL base they use.

Reply:
To be confirmed with @nvdk 
Based on choice made in https://github.com/lblod/app-gelinkt-notuleren/blob/master/config/resources/master-mobiliteit.lisp

Reply:
Aha! I didn't know where we got the current ones.